dashboard/cron.py

```diff
@@ -595,7 +595,7 @@ def do(self):
             status += f"ERROR: Those course ids are invalid: {invalid_course_id_list}\n"
             status += "End cron: " + str(datetime.now()) + "\n"
             logger.info("************ total status=" + status + "/n/n")
-            return (status,)
+            return status
```

Comment:
The "success" part of this method was returning a simple string, so I just removed the tupling here to fix the mypy error. I don't think this return value is being used anyway, but I don't know `django-cron` as well as @jonespm.

Reply:
In django-cron is just calls

`                    self.msg = self.cron_job.do()`

So a str seems more appropriate as a return. This is the value that's put into the log.